Zend/zend_constants.c

```diff
@@ -108,10 +109,9 @@ void zend_startup_constants(void)
 
 void zend_register_standard_constants(void)
 {
-	REGISTER_MAIN_LONG_CONSTANT("E_ERROR", E_ERROR, CONST_PERSISTENT | CONST_CS);
+	register_zend_constants_consts(0);
```

Comment:
I wonder whether there can be a single function call for arginfo generated definitions. I mean, in near future there will also be a function for setting parameter attributes on functions (I suppose).

Do we want our API usage to look like:
```
register_zend_constants_consts(0);
register_zend_constants_attributes(0);
register_zend_constants_some_yet_other_things(0);
```
or just a condensed
```
register_zend_constants_definitions(0); // name TBD
```
which then calls all of these generated functions.

Reply:
> I wonder whether there can be a single function call for arginfo generated definitions. I mean, in near future there will also be a function for setting parameter attributes on functions (I suppose).

Yeah, you are right! When I was thinking about how to properly support parameter attributes via gen_stubs.php (in connection with https://github.com/php/php-src/pull/8352), I also came to the conclusion: there should be a single function which could make some initialization for an extension.

However, I'd like to defer solving the above issue in order not to make this PR even larger, until implementing sensitive attributes.